pio/CMakeLists.txt

```diff
@@ -31,38 +30,40 @@ endif()
 
 # Netcdf is required
 
+SET (CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../pio2/cmake" ${CMAKE_MODULE_PATH})
```

Comment:
Yes, I think we should open a different issue on this. In this PR I am just copying over what is already in ESMCI/cime. I also don't think setting the CMAKE_MODULE_PATH to ${CMAKE_CURRENT_SOURCE_DIR}/../../pio2/cmake is the right thing to do (shouldn't the makefile set the USER_CMAKE_MODULE_PATH instead).